types/passport-cognito/index.d.ts

```diff
@@ -8,22 +8,35 @@ import passport = require('passport');
 import express = require('express');
 
 export interface CognitoStrategyOptions {
-  userPoolId: string;
-  clientId: string;
-  region: string;
+    userPoolId: string;
+    clientId: string;
+    region: string;
 }
 
 export type CognitoVerifyFunction = (
-  accessToken: string,
-  idToken: string,
-  refreshToken: string,
-  user: object,
-  done: (error: any, user?: any) => void,
+    accessToken: string,
+    idToken: string,
+    refreshToken: string,
+    user: object,
+    done: (error: any, user?: any) => void,
+) => any;
+
+export type CognitoVerifyFunctionWithSession = (
```

Comment:
This can be an interface, and can extend `CognitoVerifyFunction` to reduce duplication